index.js

```diff
@@ -12,9 +84,149 @@ function toggleMenu() {
+  event.preventDefault();
+  modalContent = {
+    header: '',
+    technologies: [],
+    description: '',
+    img: '',
+
+  };
+  document.querySelector('.modal').classList.toggle('hide');
+});
+
+function launchProject(id) {
+  if (id >= 0) {
+    modalContent.header = projects[id].name;
+    modalContent.technologies = projects[id].technologies;
+    modalContent.description = projects[id].description;
+    modalContent.img = projects[id].image;
+
+    document.querySelector('.modal').classList.toggle('hide');
+    document.querySelector('.modal-content h2').innerText = modalContent.header;
+    document.querySelector('.modal-content-body > div:first-child').style.setProperty('background-image', `url("${modalContent.img}")`);
+    let ulContent = '';
+    modalContent.technologies.forEach((item) => {
+      ulContent += `<li><span>${item}</span></li>`;
+    });
+    const ul = document.querySelector('.modal-content ul');
+    ul.innerHTML = ulContent;
+    document.querySelector('.modal-content-body p').innerText = modalContent.description;
+  }
+}
+
+const container = document.querySelector('#content');// load projects
+
+const ulTiles = document.createElement('ul');
+ulTiles.classList.add('grid-container');
+
+projects.forEach((project, index, projectArray) => {
+  if (index === 0) {
+    const img = document.createElement('img');
+    img.src = project.image;
+    img.alt = 'Image placeholder';
+    const cardLeft = document.createElement('div');
+    cardLeft.classList.add('card-left');
+    cardLeft.appendChild(img);
+
+    const h3 = document.createElement('h3');
+    h3.innerText = 'Multi-Post Stories';
+    const p = document.createElement('p');
+    p.innerText = `A daily selection of privately personalized reads; no
+     accounts or sign-ups required. has been the industry's
+     standard dummy text ever since the 1500s, when an unknown
+     printer took a standard dummy text.`;
+    const ul = document.createElement('ul');
+    project.technologies.forEach((tec, index, arr) => {
+      const li = document.createElement('li');
+      li.innerText = tec;
+      ul.appendChild(li);
+    });
+    const dv = document.createElement('div');
+    dv.appendChild(ul);
+
+    const input = document.createElement('input');
+    input.type = 'submit';
+    input.value = 'See Project';
+    input.addEventListener('click', () => launchProject(0));
+
+    const cardRight = document.createElement('div');
+    cardRight.classList.add('card-right');
+    cardRight.appendChild(h3);
+    cardRight.appendChild(p);
+    cardRight.appendChild(dv);
+    cardRight.appendChild(input);
+
+    const cardWorks = document.createElement('div');
+    cardWorks.classList.add('card-works');
+    cardWorks.appendChild(cardLeft);
+    cardWorks.appendChild(cardRight);
+
+    container.appendChild(cardWorks);
+  } else if (index === 1) {
+    const input = document.createElement('input');
+    input.type = 'submit';
```

Comment:
- [ ] Kindly make sure when hovering in the project card the following view should appear, so I can click on the see the project button to open the modal.
> ![image](https://user-images.githubusercontent.com/53402600/196638402-428b9969-24d6-4709-a504-e0908b4f23a3.png)

Reply:
@omarsalem7 , kindly note the design template is miss leading. however let me effect the change now